Migration: Corporate Billing (#1257)
aiven/devportal: docs/platform/concepts/corporate-billing.rst

Corporate billing
=================

Everyone has their preferred way of paying for services. At Aiven, we give you the flexibility to receive invoices or pay by credit card for your Aiven services.

The **default billing** method for all new Aiven customers is a credit card charge at the end of the month for all charges accrued over that month. Each Aiven project is charged separately.

The billing unit for Aiven services is by the clock hour. The charges are automatically calculated based on the services running in your Aiven project. 

> We offer corporate billing options for customers who prefer to pay by invoice. The standard corporate billing option is monthly invoices generated at the end of the month based on actual usage, similar to the credit card charge method used by default.

Reviewer: ```suggestion
We offer corporate billing options for customers who prefer to pay by invoice. The standard corporate billing option is monthly invoices generated at the end of the month based on actual usage, similar to the credit card billing method.
```

These invoices are automatically generated by our backend and sent as PDF over email to the billing email addresses configured in the Aiven dashboard.  

Prices for Aiven services are always defined in US dollars, but invoices can be sent in other currencies using the invoice date's currency exchange rates.  Currently, the following currencies are accepted:

-  US dollars (default)

-  Euros

-  Pounds sterling 

-  Swiss francs

-  Danish kroner

-  Norwegian kroner

-  Swedish kronor

-  Australian dollars 

-  Canadian dollars 

> A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction. Invoice must be paid within **10 days** of receipt by a wire transfer to Aiven bank account at Citibank Europe Plc:

Reviewer: This line is not very clear to me. Maybe rewrite it?

"A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction."

Author: I'm deleteting this line "A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction."

Corporate Billing
=================

Receive invoices instead of credit card charges for your Aiven services

The **default billing** method for all new Aiven customers is a credit card charge at the end of the month for all charges accrued over that month. Each Aiven project is charged separately.

The billing unit for Aiven services is 1 hour. We automatically calculate the amount based on the services that have been running in your Aiven project. 

> For customers who prefer to pay by invoice or are unable to pay with a credit card, we also offer corporate billing options.  The standard corporate billing option is monthly invoices generated at the end of the month based on actual usage, similar to the credit card charge method used by default.

Reviewer 2: ```suggestion
For customers who prefer to pay by invoice or are unable to pay with a credit card, we also offer corporate billing options. The standard corporate billing option generates monthly invoices at the end of the month based on actual usage, similar to the credit card charge method used by default.
```

> These invoices are automatically generated by our backend and sent as PDFs over email to the billing email addresses configured in the Aiven dashboard.  

Reviewer 2: ```suggestion
These invoices are automatically generated by our backend and sent as PDF over email to the billing email addresses configured in the Aiven dashboard.  
```

Prices for Aiven services are always defined in US dollars, but invoices can be sent in other currencies using the invoice date's currency exchange rates.  Currently, the following currencies are accepted:

-  US dollars (default)

-  Euros

-  Pounds sterling 

-  Swiss francs

-  Danish kroner

-  Norwegian kroner

-  Swedish kronor

-  Australian dollars 

-  Canadian dollars 

> Invoices must be paid within 10 days of receipt by a wire transfer to Aiven bank account at Citibank Europe Plc:

Reviewer 2: We might need to add some words to the vocabulary here
